utils/scripts/update_generated_code_from_idl.sh

```diff
@@ -40,7 +50,13 @@ for idl_file in "${idl_files[@]}"; do
 
     cd "${idl_dir}"
 
-    fastddsgen -replace "$file_from_gen"
+    # Detect if needs type_object.
+    [[ ${files_needing_typeobject[*]} =~ $idl_file ]] && to_arg='-typeobject' || to_arg=''
+
+    # Detect if needs case sensitive.
+    [[ ${files_needing_case_sensitive[*]} =~ $idl_file ]] && to_arg='-cs' || to_arg=''
```

Comment:
This instruction is overwritting the previous one, so the TypeObject is not being regenerated for any of the listed IDLs.

Reply:
Done in 67797b9